api/tests/identity-access-management/integration/infrastructure/repositories/anonymous-user-token.repository.test.js

```diff
@@ -0,0 +1,51 @@
+import { anonymousUserTokenRepository } from '../../../../../src/identity-access-management/infrastructure/repositories/anonymous-user-token.repository.js';
+import { temporaryStorage } from '../../../../../src/shared/infrastructure/key-value-storages/index.js';
+import { expect } from '../../../../test-helper.js';
+
+const anonymousUserTokenTemporaryStorage = temporaryStorage.withPrefix('anonymous-user-token:');
+
+describe('Integration | Identity Access Management | Infrastructure | Repository | anonymous-user-token', function () {
```

Comment:
Je ne sais pas si nous avons standardisé exactement quel nom donner à un test sur un repository, mais la suggestion ci-dessous me semble plus adaptée que le nom présent actuellement dans la PR : 
```suggestion
describe('Integration | Identity Access Management | Infrastructure | Repository | anonymousUserTokenRepository', function () {
```

Reply:
Je ré-ouvre cette discussion car le point n’a pas été modifié..

Reply:
Ah désolé ! un oubli, c'est fait :)